home/graphql/queries/ProfileProjectsQuery.query.graphql

```diff
@@ -2,18 +2,8 @@ query Home_MyApps($limit: Int!, $offset: Int!) {
   me {
     id
     appCount
-    apps(limit: $limit, offset: $offset) {
-      id
-      description
-      fullName
-      iconUrl
-      lastPublishedTime
-      name
-      username
-      packageName
-      privacy
-      sdkVersion
-      published
+    apps(limit: $limit, offset: $offset, includeUnpublished: true) {
```

Comment:
If the overarching intent is to always include unpublished projects, wouldn't we want to change the server to ignore the includeUnpublished parameter so that the concept of being "published" holds no meaning?

Reply:
Tried that in https://github.com/expo/universe/pull/8403, had to undo it in https://github.com/expo/universe/pull/8457. The problem is mentioned in the "how" section of this PR, but essentially you can't query the `lastPublishedTime` field if you use `includeUnpublished` (or we ignore it as I did in that PR), and old clients always query `lastPublishedTime`.

Reply:
One alternative option would be to deprecate the `apps` field on User/Account and make an `apps2` that includes unpublished by default. I don't feel strongly one way or the other though.

`lastPublishedTime` is already deprecated.

Reply:
Not to force things too much but would it (a) work and (b) be maintainable if we had logic like:
```
// on server
query.includeUnpublished = !query.lastPublishedTime
```
?
That is, if you query lastPublishedTime, you only get "published" apps.

Reply:
Seems a bit fragile to me. I'd rather just deprecate and replace. And we should just have `lastPublishedTime` be nullable since it is nullable.